Schemas/Bulk/Ed-Fi-Core.xsd

```diff
@@ -7466,7 +7471,7 @@
           </xs:element>
           <xs:element name="StudentParticipationCode" type="StudentParticipationCodeDescriptorReferenceType">
             <xs:annotation>
-              <xs:documentation>The role or type of participation of a student in a discipline incident; 
+              <xs:documentation>The role or type of participation of a student in a discipline incident;
```

Comment:
This change is showing up in multiple PRs, which ticket is it supposed to be linked to?  Is this just some white space clean up on these docs?

Reply:
It's white space clean up. Rather than have to rebase each subsequent pr, I just left the change in all of them.